meshroom/ui/qml/main.qml

```diff
@@ -482,6 +494,20 @@ ApplicationWindow {
         onTriggered: graphEditor.pasteNodes()
     }
 
+    Action {
+        id: loadTemplateAction
+
+        property string tooltip: "Load a template like a regular project file (any \"Publish\" node will be displayed)"
+        text: "Load Template"
+        shortcut: "Ctrl+Shift+O"
```

Comment:
Maybe avoid shortcuts for so specific options.
```suggestion
```

Reply:
Done, the shortcut has been removed.